nautobot/project-static/js/forms.js

```diff
@@ -226,10 +226,7 @@ function jsify_form(context) {
                         // Annotate hierarchical depth for MPTT objects
                         record.text = '--'.repeat(record._depth) + ' ' + record.text;
                     }
-                    if (record.tree_depth) {
-                        // Annotate hierarchical depth for django-tree-queries objects
-                        record.text = '--'.repeat(record.tree_depth) + ' ' + record.text;
-                    }
```

Comment:
@christhant With this removal it'll make the LocationType dropdown selection harder to parse

<img width="761" alt="Screen Shot 2022-09-06 at 10 25 18" src="https://user-images.githubusercontent.com/31187/188661756-1cce7ba7-24d8-4c13-b7fb-c2cea447746f.png">

Therefor I think we should implement a similar `display` property on LocationType as well.